strictdoc/backend/sdoc_source_code/models/source_node.py

```diff
@@ -13,16 +13,24 @@
 from strictdoc.backend.sdoc_source_code.models.range_marker import (
     RangeMarker,
 )
+from strictdoc.backend.sdoc_source_code.models.source_location import ByteRange
 from strictdoc.core.project_config import SourceNodesEntry
 
 
-@dataclass
+@dataclass(eq=False)
 class SourceNode:
+    """
+    NOTE: eq=False is needed to make this dataclass support being a dictionary key.
+    """
+
     entity_name: Optional[str]
+    byte_range: ByteRange
```

Comment:
A SourceNode can have multiple markers, meaning it can cover multiple *non-consecutive* ranges. However, there's only a singe `byte_range` attribute here, which can cover only a *single consecutive* range. What range is it meant to cover? Everything from start of first marker to end of last marker?

Reply:
Renamed to comment_byte_range.